cdc/tests/.unmodify/multi_capture/run.sh

```diff
@@ -20,41 +21,25 @@ function run() {
 
 	# record tso before we create tables to skip the system table DDLs
 	start_ts=$(run_cdc_cli_tso_query ${UP_PD_HOST_1} ${UP_PD_PORT_1})
+	sleep 10
 
-	# create $DB_COUNT databases and import initial workload
-	for i in $(seq $DB_COUNT); do
-		db="multi_capture_$i"
-		run_sql "CREATE DATABASE $db;"
-		go-ycsb load mysql -P $CUR/conf/workload1 -p mysql.host=${UP_TIDB_HOST} -p mysql.port=${UP_TIDB_PORT} -p mysql.user=root -p mysql.db=$db
-	done
+	rawkv_op $UP_PD put 10000
 
 	# start $CDC_COUNT cdc servers, and create a changefeed
 	for i in $(seq $CDC_COUNT); do
-		run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --logsuffix "$i" --addr "127.0.0.1:830${i}"
+		run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --logsuffix "$i" --addr "127.0.0.1:860${i}"
 	done
 
-	TOPIC_NAME="ticdc-multi-capture-test-$RANDOM"
 	case $SINK_TYPE in
-	kafka) SINK_URI="kafka://127.0.0.1:9092/$TOPIC_NAME?protocol=open-protocol&partition-num=4&kafka-version=${KAFKA_VERSION}&max-message-bytes=10485760" ;;
-	*) SINK_URI="mysql://normal:123456@127.0.0.1:3306/" ;;
+	tikv) SINK_URI="tikv://${DOWN_PD_HOST}:${DOWN_PD_PORT}" ;;
+	*) SINK_URI="" ;;
 	esac
-	run_cdc_cli changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI"
-	if [ "$SINK_TYPE" == "kafka" ]; then
-		run_kafka_consumer $WORK_DIR "kafka://127.0.0.1:9092/$TOPIC_NAME?protocol=open-protocol&partition-num=4&version=${KAFKA_VERSION}&max-message-bytes=10485760"
-	fi
 
-	# check tables are created and data is synchronized
-	for i in $(seq $DB_COUNT); do
-		check_table_exists "multi_capture_$i.USERTABLE" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
-	done
-	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
+	run_cdc_cli changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI"
```

Comment:
Suggest to create plenty of changfeeds to utilize all the captures.

Reply:
Added.